app/api/database/Holders/index.js

```diff
@@ -0,0 +1,37 @@
+import { db } from 'configs/firebase'
```

Comment:
I think we should rename the other folders to reflect the name of the collections (`Patients`, `Payments`, `Holders`) with first capital letter because it represents an "Entity". What do you think?